docs/_templates/footer.html

```diff
@@ -0,0 +1 @@
+Anthony Ozerov, Jeffrey Smith, and the NASA ATAP team.
```

Comment:
Not necessary to complete this merge, but we should add an authors and citations page to the documentation where we list the authors in more details (affiliation, ORCID, maybe even short bio). We should also add a citation request for anyone who wants to use our package or data, and give a bibtex example of how to cite our work. That should include both the Icarus paper I wrote and some citation for your package.